### PR TITLE
Added _394.java

### PR DESCRIPTION
**Pull request by varunu28**
- [X] Added code with description
 - [X] Added test case
 - [X] Updated README

### CODE REVIEW

**Comment by fishercoder1534** [comment]
Left some comments, LGTM otherwise, thanks!

**Comment by varunu28** [comment]
@fishercoder1534 I am unable to view your comments on the PR and it is showing me no reviews against the PR also. I was facing this issue during my last PR too. Not sure if Github changed something.

**Comment by fishercoder1534** [comment]
Looks like so, I think the problem was: I needed to use "Request Changes" and then click "Submit" instead of just commenting on the code. Lessons learned, fun!

File: src/main/java/com/fishercoder/solutions/_394.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
+package com.fishercoder.solutions;
+
+import java.util.Stack;
+
+public class _394 {
+
+    public static class Solution {
```
[the remark this answers is not in the document]
Comment (fishercoder1534) [reply src/main/java/com/fishercoder/solutions/_394.java]: No, it's unique. It's not a namespace conflict thing, we usually start with `Solution1`, although there might be the only solution for that problem at the moment. 
If you could stick to this convention, that'd be great.

[the remark this answers is not in the document]
Comment (varunu28) [reply src/main/java/com/fishercoder/solutions/_394.java]: Sounds great. Would make the changes

File: src/main/java/com/fishercoder/solutions/_394.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                    StringBuilder sb = new StringBuilder();
+                    int n = count.pop();
+
+                    for (int j = 0; j < n; j++) {
+                        sb.append(st);
+                    }
+
+                    str.push(str.pop() + sb.toString());
+                }
+                else {
```
[the remark this answers is not in the document]
Comment (varunu28) [reply src/main/java/com/fishercoder/solutions/_394.java]: Done

**Comment by fishercoder1534** [comment]
Tests are failing: https://travis-ci.org/fishercoder1534/Leetcode/builds/478155671?utm_medium=notification&utm_source=email

Could you fix it? @varunu28

**Comment by fishercoder1534** [comment]
All right, I fixed it, please run your tests before you submit next time.

**Pull request merged**